config/webpack.plugins.js

```diff
@@ -38,6 +38,7 @@ const plugins = (dev = false, beta = false) => {
         { 'react-dom': { singleton: true, eager: true, requiredVersion: deps['react-dom'] } },
         { 'react-router-dom': { singleton: true, requiredVersion: deps['react-router-dom'] } },
         { 'react-redux': { requiredVersion: deps['react-redux'] } },
+        { '@openshift/dynamic-plugin-sdk': { singleton: true, requiredVersion: deps['@openshift/dynamic-plugin-sdk'] } },
```

Comment:
These `requiredVersion: package.dependencies['module-name']` bits (here and above/below) might not be necessary.

AFAIK webpack module federation plugin code already attempts to auto-detect this information from dependencies in the project's `package.json` file.

Reply:
I think you might be right. We should take a look at it during our next tech debt sweep.